converters/toZigbee.js

```diff
@@ -2037,8 +2037,9 @@ const converters = {
     danfoss_mounted_mode_control: {
         key: ['mounted_mode_control'],
         convertSet: async (entity, key, value, meta) => {
-            const payload = {'danfossMountedModeControl': {value: (value ? 0x00: 0x01), type: 0x10}};
+            const payload = {'danfossMountedModeControl': (value ? 0 : 1)};
             await entity.write('hvacThermostat', payload, manufacturerOptions.danfoss);
+            return {readAfterWriteTime: 200, state: {'mounted_mode_control': value}};
```

Comment:
Do we want to return 0 and 1 here or true and false?

I also just checked: `danfossMountedModeControl: {ID: 0x4013, type: DataType.boolean, manufacturerCode: ManufacturerCode.DANFOSS},`

We probably want to take a bool as imput and just pass true or false as value here.

Reply:
@sjorge Apologies if I might sound dumb but not sure how would you like me to approach this.

One thing to note is that danfossMountedModeControl specs is:
```
0x00 Go to mounting mode (the eTRV can now be
mounted on a valve)
0x01 Go to Mounted posittion (the eTRV now act as if it's
mounted on a valve)
```

I guess for heat_available this code:
```
const payload = {'danfossHeatAvailable': (value ? 1: 0)};
            await entity.write('hvacThermostat', payload, manufacturerOptions.danfoss);
```
Can be simplified to this:
```
await entity.write('hvacThermostat', {'danfossHeatAvailable': value}, manufacturerOptions.danfoss);
```

Is this what you are referring to?

Reply:
Correct, as this is a bool value we can further reduce it to just the simplified version.

Reply:
@sjorge Hi, had to disappear earlier so I couldn't leave a comment despite pushing a commit for this. Please see https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/293cef0be63c9ae7ba4bc853581dca14271c07c7

Now that works fine, but the false/true states are exact opposites of what the specification says. I have updated the description in the devices but I am not sure I should do that. What is your thoughts?

PS: I have tried to invert the values instead of description but somehow failed. I was however in a rush so maybe I did oversee sth

Reply:
So when setting `danfossMountedModeControl` to true, `danfossMountedModeActive` will also turn to true because it can be mounted. Once you mount it both go to false?

Or if you don't mount it and send false to `danfossMountedModeControl`, then `danfossMountedModeActive` transitions to false?

Wait... or is it the other way around, the description is a bit weird for those. I'd say just pass true/false as is to both without inverting.

Reply:
Yes, they should always be the same value. Active gets reported but Control is not. You need to read it to get a value. Also Active is not writable, only Control is.

This is with the TRV mounted on the radiator
![image](https://user-images.githubusercontent.com/9019306/119225654-e3204580-bafc-11eb-8fe8-b29e2528aa07.png)

I agree that the description in the specs might be wrong, at least based on what I am getting from the TRV response, maybe it is an oversight on their side.

No further changes to this code